modules/ROOT/pages/add-api-to-unified-schema.adoc

```diff
@@ -105,6 +106,16 @@ If you need to edit the URL of an API that you've added to the unified schema, y
 . Type in the new URL.
 . Click *Save*.
 
+[[change-the-authentication-method-for-an-api-aded-to-the-unified-schema]]
```

Comment:
@mauro-dentoni these are the steps to change auth method when you edit an API

Reply:
Looks fine. With the current behavior, the only way of editing the configuration of an authentication of an API is by removing the configuration and adding a new one.